### PR TITLE
Stabilize Video - New functionality

### PR DESCRIPTION
**Pull request by gpantelis**
New method that stabilaze a video

### CODE REVIEW

**Comment by Zulko** [comment]
Thanks for this, seems very useful, can you change ``stabilaze->stabilize`` and make the code PEP8 compatible (4-spaces indent, one space after commas) ?

File: moviepy/video/io/ffmpeg_tools.py [hunk 1]
```diff
@@ -66,4 +66,30 @@ def ffmpeg_resize(video,output,size):
...
+    which is more stable
+
+    Parameters
+    -----------
+
+    filename
+    The name of the shaky video
+
+    output
+    The name of new stabilazed video
```
Comment (mbeacom) [inline moviepy/video/io/ffmpeg_tools.py]: Can you rename `stabilazed` to `stabilized` here.

File: moviepy/video/io/ffmpeg_tools.py [hunk 2]
```diff
@@ -66,4 +66,30 @@ def ffmpeg_resize(video,output,size):
...
+    """
+
+    name, ext = os.path.splitext(filename)
+    if not output:
+        output = "%s_stabilazed%s"%(name, ext)
```
Comment (mbeacom) [inline moviepy/video/io/ffmpeg_tools.py]: Can you please rename `stabilazed` to `stabilized` here.  I was going to say, we might want to elect to use the new style string formatting as well `output = '{0}_stabilized{1}.format(name, ext)`, but it seems we're using the old style throughout the file, so: `output = %s_stabilized%s" % (name, ext)` should be fine.

File: moviepy/video/io/ffmpeg_tools.py [hunk 3]
```diff
@@ -66,4 +66,30 @@ def ffmpeg_resize(video,output,size):
...
+    Parameters
+    -----------
+
+    filename
+    The name of the shaky video
+
+    output
+    The name of new stabilazed video
+    """
+
```
Comment (mbeacom) [inline moviepy/video/io/ffmpeg_tools.py]: This return can be removed to align with PEP-257.

File: moviepy/video/io/ffmpeg_tools.py [hunk 4]
```diff
@@ -66,4 +66,30 @@ def ffmpeg_resize(video,output,size):
              output]
              
     subprocess_call(cmd)
+    
+def ffmpeg_stabilize_video(filename, output = None):
+    """
+
```
Comment (mbeacom) [inline moviepy/video/io/ffmpeg_tools.py]: This return can be removed.

File: moviepy/video/io/ffmpeg_tools.py (hunk 2 above)
Comment (gpantelis) [inline moviepy/video/io/ffmpeg_tools.py]: So you want the new or the old style?

Comment (mbeacom) [inline moviepy/video/io/ffmpeg_tools.py]: The old style should be sufficient, as it'll conform to the styling throughout this file for now.

**Comment by gpantelis** [comment]
What im doing wrong?

**Comment by Gloin1313** [comment]
I think we should have a test for this in /tests. Also, I'd be interested to see a before and after video (or 2 separate videos) of this PR being used.

**Comment by gpantelis** [comment]
Okay I'm working on it!

**Comment by earney** [comment]
@gpantelis 

here is some code that could help you...

```

ffmpeg_stabilized_video("deshake_test.mp4", output="deshake_output.mp4")

video=VideoFileClip("deshake_test.mp4")
video1=VideoFileClip("deshake_output.mp4")
video.size=(video.w/2, video.h)
video1.size=(video1.w/2, video1.h)

video=clips_array([[video, video1]])
video.write_videofile("deshake.mp4")
```

**Comment by gpantelis** [comment]
Nice thanks!

**Comment by kencochrane** [comment]
@gpantelis thanks for the PR, just curious if you had a chance to work on the unit tests yet? Need any help?

**Comment by tburrows13** [comment]
I'm ready for this to be merged.

**Pull request merged**